nnvm/tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -116,13 +116,25 @@ def compare_tf_with_tvm(in_data, in_name, out_name, init_global_variables=False)
             if not ctx.exist:
                 print("Skip because %s is not enabled" % device)
                 continue
+            if no_gpu and device == 'cuda':
+                continue
 
             tvm_output = run_tvm_graph(final_graph_def, in_data,
                                        in_node, tf_output.shape, tf_output.dtype, target=device)
             np.testing.assert_allclose(tf_output, tvm_output, atol=1e-5, rtol=1e-5)
 
         sess.close()
 
+def is_gpu_available():
+    from tensorflow.python.client import device_lib
+    local_device_protos = device_lib.list_local_devices()
+    gpu_list = [x.name for x in local_device_protos if x.device_type == 'GPU']
+    if len(gpu_list) < 0:
+        print("Tensorflow GPU:", gpu_list)
```

Comment:
Need this print?

Reply:
I left it intentionally to print the GPU details.